<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LTSView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ltsview.qrc">
    <normaloff>:/ltsview/icons/ltsview.ico</normaloff>:/ltsview/icons/ltsview.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="exportBitmap"/>
     <addaction name="exportText"/>
    </widget>
    <addaction name="open"/>
    <addaction name="openTrace"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="resetViewpoint"/>
    <addaction name="separator"/>
    <addaction name="zoomIntoAbove"/>
    <addaction name="zoomIntoBelow"/>
    <addaction name="zoomOut"/>
    <addaction name="separator"/>
    <addaction name="displayStates"/>
    <addaction name="displayTransitions"/>
    <addaction name="displayBackpointers"/>
    <addaction name="displayWireframe"/>
    <addaction name="separator"/>
    <addaction name="preferences"/>
    <addaction name="separator"/>
    <addaction name="actionShowGraphicsInfo"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="informationDock">
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="simulationDock">
   <property name="windowTitle">
    <string>Simulation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="markDock">
   <property name="windowTitle">
    <string>Marks</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="settingsDock">
   <property name="windowTitle">
    <string>Settings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="openTrace">
   <property name="text">
    <string>Open &amp;Trace...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="exportBitmap">
   <property name="text">
    <string>&amp;Bitmap...</string>
   </property>
  </action>
  <action name="exportText">
   <property name="text">
    <string>&amp;Text...</string>
   </property>
  </action>
  <action name="resetViewpoint">
   <property name="text">
    <string>&amp;Reset Viewpoint</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="zoomIntoAbove">
   <property name="text">
    <string>Zoom Into &amp;Above</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="zoomIntoBelow">
   <property name="text">
    <string>Zoom Into &amp;Below</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="zoomOut">
   <property name="text">
    <string>Zoom &amp;Out</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="displayStates">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;States</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="displayTransitions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;Transitions</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="displayBackpointers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Back&amp;pointers</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="displayWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display &amp;Wireframe</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="preferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionShowGraphicsInfo">
   <property name="text">
    <string>Show graphics info</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ltsview.qrc"/>
 </resources>
 <connections/>
</ui>
